src/classes/GE_GiftEntryController.cls

```diff
@@ -210,6 +210,16 @@ public with sharing class GE_GiftEntryController {
         return captureGroup;
     }
 
+    @AuraEnabled
+    public static void processPaymentsFor(Id batchId) {
+       try {
+           NPSP_Batch batch = new NPSP_Batch(batchId);
+           batch.processPayments();
+       } catch (AuraHandledException ex) {
+          throw new AuraHandledException(ex.getMessage());
+       }
```

Comment:
This should catch a regular Exception and then throw a new AuraHandled

} catch (Exception ex) {
          throw new AuraHandledException(ex.getMessage());
       }